### ISSUE TITLE
Question : move model to another module

### ISSUE THREAD

**Issue by rousseldenis** [OCA/openupgradelib#101]
I want to split some models definition in different new modules.

I don't find any openupgrade function that allows to move a model to another module. I think there is an entry in ir_model_data that associates model and module. Is the entry deleted on new module install?

Maybe there is nothing to do... Is there anybody faced the same situation

**Comment by StefanRijnhart** [comment]
I don't think we cover this case. @hbrunn fixed this for field entries though: https://github.com/OCA/OpenUpgrade/blob/11.0/odoo/addons/base/ir/ir_model.py#L761-L800 (in odoo/models.py in earlier versions). Maybe we need something similar for models indeed.

**Comment by pedrobaeza** [comment]
We only have the `update_module_names`, but it's for whole models of the module. You can take the code from this function for applying specific things for a model and create a new function `update_model_module` or similar name and factor out this code to be called isolate or from the `update_module_names` function.
